app/assets/javascripts/discourse/lib/to-markdown.js.es6

```diff
@@ -0,0 +1,211 @@
+import parseHTML from 'discourse/helpers/parse-html';
+
+String.prototype.trimLeft = function() {
```

Comment:
You should not modify the string prototype in this class. It will mean there's a side effect of including this file in the global scope.

If you need a trimLeft function just make it local to this file.